config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -10,6 +10,25 @@ spec:
     - configure-os
     - run-os
   nodeTemplate:
+    nodes:
+      edpm-compute-0:
+        ansibleHost: 172.20.12.67
+        ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+        ansibleUser: root
+        ansibleVars:
+          ansible_ssh_transfer_method: scp
+          ctlplane_ip: 172.20.12.67
+          external_ip: 172.20.12.76
+          fqdn_internal_api: edpm-compute-1.example.com
+          internal_api_ip: 172.17.0.101
+          storage_ip: 172.18.0.101
+          tenant_ip: 172.10.0.101
```

Comment:
it woudl be really nice if we coudl mvoe awary form haging to specify this in the the AnsibelVArs isntead.

we seshould be able ot generate this form the network attachments.